loopabull/plugins/rkdirectorytranslator.py

```diff
@@ -0,0 +1,27 @@
+#
+# Loopabull sub-directory playbook hierarchy
+#
+import os
+
+from loopabull.plugin import Plugin
+
+class RkdirectoryTranslator(Plugin):
+    """
+    Loopabull PTranslator to convert routing_keys to OS independent file paths.
```

Comment:
I'm guessing `PTranslator` is just a typo of some sort, but not certain ... what were you going for here?